src/Compilers/CSharp/Portable/Binder/SubsumptionDiagnosticBuilder.cs

```diff
@@ -129,6 +129,8 @@ private ErrorCode CheckSubsumed(BoundPattern pattern, DecisionTree decisionTree,
                         // the input can no longer be null. In that case a null pattern is subsumed.
                         if (isNull && !inputCouldBeNull)
                         {
+                            // Note: we do not have any test covering this. Is it reachable?
```

Comment:
I don't have a belief that this code is unreachable. Having said that, I don't know how to reach it. I'd rather not find out by breaking customer code, when the correct behavior is obvious.